integration/baggageclaim/privileges_test.go

```diff
@@ -57,16 +83,19 @@ var _ = Describe("Privileges", func() {
 		Expect(err).NotTo(HaveOccurred())
 
 		dataFilename = writeData(baseVolume.Path())
+		checkMode(filepath.Join(baseVolume.Path(), dataFilename), mode)
 
-		stat, err := os.Stat(filepath.Join(baseVolume.Path(), dataFilename))
-		Expect(err).NotTo(HaveOccurred())
-
-		Expect(stat.Mode().String()).To(Equal(mode.String()))
+		linkSentinel = makeSentinel(baseVolume.Path())
+		checkMode(linkSentinel, sentinelMode)
 	})
 
 	AfterEach(func() {
 		runner.Stop()
 		runner.Cleanup()
+
+		checkMode(linkSentinel, sentinelMode)
```

Comment:
It seems kind of non-obvious that this should be something done for every test in the file - could it be isolated to a single, new `It` instead?